src/index.html

```diff
@@ -4,10 +4,229 @@
+          <img
+            class="gallery__photo"
+            src="images/photos/4.jpg"
+            alt="House photo">
+          <h3 class="gallery__itemName">
+            <span class="gallery__itemName gallery__itemName--color">
+              Edition Residens
+            </span>
+            / Miami Beach, FL 33139
+          </h3>
+        </div>
+        <div class="gallery__item">
+          <img
+            class="gallery__photo"
+            src="images/photos/5.jpg"
+            alt="House photo">
+          <img
+            class="gallery__photo"
+            src="images/photos/6.jpg"
+            alt="Beach photo">
+          <h3 class="gallery__itemName">
+            <span class="gallery__itemName gallery__itemName--color">
+              Faena House
+            </span>
+            / Miami Beach, FL 33140
+          </h3>
+        </div>
+        <a
+          class="gallery__button button button--secondary"
+          href="#">
+          View all
+        </a>
+      </section>
+      <section
+        class="testimonials"
+        id="testimonials">
+        <h2 class="sectionTitle">Testimonials</h2>
+        <div class="testimonials__content">
+          <div class="testimonials__card">
+            <img
+              class="testimonials__photo"
+              src="./images/people/1.jpg"
+              alt="Client photo">
+            <h3 class="testimonials__name">
+              Leslie Mckinney
+            </h3>
+            <p class="testimonials__work">
+              CEO
+            </p>
+            <p class="testimonials__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+          <div class="testimonials__card">
+            <img
+              class="testimonials__photo"
+              src="./images/people/2.jpg"
+              alt="Client photo">
+            <h3 class="testimonials__name">
+              Scarlett Hawkins
+            </h3>
+            <p class="testimonials__work">
+              Senior Vice President
+            </p>
+            <p class="testimonials__text">
+              My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my hear
+            </p>
+          </div>
+          <div class="testimonials__card">
+            <img
+              class="testimonials__photo"
+              src="./images/people/3.jpg"
+              alt="Client photo">
+            <h3 class="testimonials__name">
+              Max Henry
+            </h3>
+            <p class="testimonials__work">
+              Chief Design Officer
+            </p>
+            <p class="testimonials__text">
+              When my lovely valley of steam rises and half-day sun is above an impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+      <section></section>
+    </main>
+    <footer class="page__footer">
+      <img
+          class="logo logo--size--forFooter"
```

Comment:
```suggestion
    <footer class="page__footer footer">
      <img
          class="logo footer__logo"
```
`logo--size--forFooter` looks wierd